app/views/layouts/_topbar.html.haml

```diff
@@ -14,7 +14,7 @@
     %ul.right
       %li.active
         - if user_signed_in?
-          %a{href: "#", id: "#{current_user.id}", class: "username"}= "Signed in as #{current_user.username}"
+          %a{href: "users/#{current_user.id}", id: "#{current_user.id}", class: "username"}= "Signed in as #{current_user.username}"
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace<br>`= "..."` should be rewritten as `...`<br>Avoid defining `class` in attributes hash for static class names<br>Line is too long. [132/80]